osx/docs/JenkinsWelcomePanel.rtf

```diff
@@ -1,19 +1,15 @@
-{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf350
-{\fonttbl\f0\fswiss\fcharset0 ArialMT;}
-{\colortbl;\red255\green255\blue255;}
-{\*\listtable{\list\listtemplateid1\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid1}}
-{\*\listoverridetable{\listoverride\listid1\listoverridecount0\ls1}}
+{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf810
```

Comment:
What's with all the markup here, @oleg-nenashev ?   The output when I view it seems to be more or less plaintext in appearance, is all this strictly needed?